WalletWasabi/JsonConverters/OwnershipProofJsonConverter.cs

```diff
@@ -0,0 +1,30 @@
+using NBitcoin;
+using Newtonsoft.Json;
+using System;
+using WalletWasabi.Crypto;
+
+namespace WalletWasabi.JsonConverters
+{
+	public class OwnershipProofJsonConverter : JsonConverter
+	{
+		/// <inheritdoc />
+		public override bool CanConvert(Type objectType)
+		{
+			return objectType == typeof(OwnershipProof);
+		}
+
+		/// <inheritdoc />
+		public override object ReadJson(JsonReader reader, Type objectType, object existingValue, JsonSerializer serializer)
+		{
+			var value = (string)reader.Value;
+			return OwnershipProof.FromBytes(ByteHelpers.FromHex(value));
+		}
+
+		/// <inheritdoc />
+		public override void WriteJson(JsonWriter writer, object value, JsonSerializer serializer)
+		{
+			var bytes = ((OwnershipProof)value).ToBytes();
+			writer.WriteValue(ByteHelpers.ToHex(bytes));
```

Comment:
I know i'm like a broken record about how JSON sucks, and that this is consistent with the rest of the codebase, but taking mostly pseudorandom bytes, converting them to hex and then putting them in JSON is more than doubles the bandwidth requirement...

Since we'll be sticking with JSON, perhaps we should at least use base64 to make that a little less inefficient?

TBF this is mostly an issue for the range proofs, not this data but I'm a little triggered because of my JSON phobia and I can't help but comment ;-)

Reply:
We don't need to stick to json. Json makes more sense when the data can be read by humans but in this case absolutely everything looks like random bytes.
Lets implement it using json first as we agreed and then once we achieve our goal (or before that if we have time) try with something more efficient.

Reply:
IIRC dropping JSON was vetoed, so assuming using it is a requirement JSON, binary data as base64 makes more sense to me than hex. Anyway, it's not important right now.